composer.json

```diff
@@ -23,8 +23,7 @@
     "exclude-from-classmap": []
   },
   "config": {
-    "preferred-install": "dist",
-    "prepend-autoloader": false
```

Comment:
Any reason to remove prepend-autoloader?

Reply:
@jolelievre